docs/api/en/materials/MeshMatcapMaterial.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		[page:Material] &rarr;
+
+		<h1>[name]</h1>
+
+		<p class="desc">
+			Represents a (complete) material including lighting and reflections. Hence, it does not react on lights.
```

Comment:
I'm not sure what a complete material is... we need a better definition...

Reply:
My best attempt:

`MeshMatcapMaterial` is defined by a _MatCap_ (or _Lit Sphere_) texture, which encodes the material color and shading.

Reply:
> I'm not sure what a complete material is... we need a better definition...

I've used this term since I've read it multiple times in the literature. Anyway, your suggestion is more descriptive so let's go with it.